addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -181,7 +181,7 @@ const Wysiwyg = Widget.extend({
             commands: commands,
             onChange: options.onChange,
             plugins: options.editorPlugins,
-            direction: localization.direction || 'ltr',
```

Comment:
Just passing by (did not check the rest): normal this line is removed?

Reply:
Nope, it was a mistake because of a cherry pick with conflict wrongly resolved.
Thank you for having spotted it.